.gitlab-ci.yml

```diff
@@ -80,7 +80,7 @@ build-prod-image:
     - docker buildx build
       --label target=staging
       --build-arg CI_COMMIT_SHA=${CI_COMMIT_SHA}
-      --tag registry.ddbuild.io/lemur:v$CI_PIPELINE_ID-$CI_COMMIT_SHORT_SHA
+      --tag registry.ddbuild.io/lemur:$CI_COMMIT_TAG
```

Comment:
On line 87 below we sign the image with the `$CI_COMMIT_TAG` so this must match:
```
ddsign sign registry.ddbuild.io/lemur:$CI_COMMIT_TAG --docker-metadata-file ${METADATA_FILE}
```